apps/src/templates/projects/NewProjectButtons.jsx

```diff
@@ -121,7 +121,7 @@ const PROJECT_INFO = {
   music: {
     label: i18n.projectTypeMusic(),
     thumbnail: studio('/shared/images/courses/logo_music.png'),
-    urlOverride: '/s/music-intro-2024/reset',
+    urlOverride: 'https://code.org/music',
```

Comment:
Can we use `pegasus()` (similar to how `studio()` is used above) so that it links to the environment-specific site?

Reply:
Good idea. Done.